x-pack/plugins/spaces/server/saved_objects/schemas.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+
+export const SpacesSavedObjectSchemas = {
```

Comment:
__nit:__ Can we add unit tests for this schema? Similar to the tests for `x-pack/plugins/spaces/server/lib/space_schema.ts` (they will be simpler since the SO schema is simpler that the schema we use for routes, we basically need to check for types and "required" condition).

Reply:
Added in latest commit!